### PR TITLE
chore: remove deprecated usage of "io/ioutil"

### PR DESCRIPTION
**Pull request by cesarvspr**

### CODE REVIEW

**Comment by cesarvspr** [comment]
closes https://github.com/nats-io/nats.go/issues/1149

File: go.mod [hunk 1]
```diff
@@ -3,6 +3,9 @@ module github.com/nats-io/nats.go
 go 1.16
 
 require (
+	github.com/golang/protobuf v1.5.2
+	github.com/nats-io/nats-server/v2 v2.9.8
```
Comment (wallyqs) [inline go.mod:7]: thanks @cesarvspr for the PR! You can avoid changing both `go.mod` and `go.sum` by running the tests as follows:
```
go test -modfile=go_test.mod ./...
```

**Review by piotrpio — COMMENTED** [review]
Thanks for the contribution! As per @wallyqs comment, please clean up `go.mod` and `go.sum` (they should not contain test-only dependencies), then LGTM 👍

File: go.mod (hunk 1 above)
Comment (cesarvspr) [reply go.mod]: This works, thanks! Any idea why [tests are intermittent](https://github.com/nats-io/nats.go/runs/9834152071) on the pipeline? @wallyqs

**Review by piotrpio — APPROVED** [review]
LGTM!

**Pull request merged**